modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeJavaGetLayoutDisplayPageObjectProviderCheck.testjava

```diff
@@ -61,5 +61,6 @@ public class UpgradeJavaGetLayoutDisplayPageObjectProviderCheck {
 	private final LayoutDisplayPageProviderRegistry
 		_layoutDisplayPageProviderRegistry;
 	private final Portal _portal;
+	private LayoutDisplayPageProvider layoutDisplayPageProvider;
```

Comment:
Hey @TamyrisTorres, the commits should be atomic, e.g. they should do a unique action, in light of this the test fixes, ideally, should be in another commit.